wgengine/router/router_linux.go

```diff
@@ -511,33 +511,13 @@ func (r *linuxRouter) delRoute(cidr netaddr.IPPrefix) error {
 // addSubnetRule adds a netfilter rule that allows traffic to flow
 // from Tailscale to cidr.
 func (r *linuxRouter) addSubnetRule(cidr netaddr.IPPrefix) error {
```

Comment:
Now kinda useless since this function is empty(?), but maybe 'cidr' should be destCIDR or something for clarity. Same below for the del version.

Reply:
Yup, all the subnet route stuff is unnecessary now. I'm cleaning that up now, and the tests. Yay deleting code!